.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
This will keep new files in target from being stored in git, but there are some .class files that are already in there and should be removed.

Also, the entire /test-ouput/ directory should probably be ignored as well